src/_gettsim/sozialversicherung/rente/grundrente/grundrente.py

```diff
@@ -204,22 +143,7 @@ def basisbetrag_m(
 def durchschnittliche_entgeltpunkte(
```

Comment:
I have to stop now, but we really should not have one function (input) `mean_entgeltpunkte` and another one `durchschnittliche_entgeltpunkte`. I believe we settled on the `mean_` convention.